### PR TITLE
Add ability to set a global refreshToken when DMing the bot

### PR DESCRIPTION
**Pull request by who03**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -24,6 +25,16 @@ client.on('message', message => {
       }
     )
   }
+  if (message.author.username != client.user.username) {
+    if (message.channel.type === "dm") {
+        const msg = message.content.split(" ")
+        if (msg[0] === "setRefresh" && msg.length > 1) {
+          refreshToken = msg[1]
+          console.log("Refresh Token set to: " + refreshToken)
```
Comment (Laturine) [inline app.js]: let's do 'Refresh Token set to: ${refreshToken}'

File: app.js [hunk 2]
```diff
@@ -14,20 +15,26 @@ client.on('message', message => {
...
+      resp => {
+        let teamId = nbaUtil.getTeamId(resp, firstName, lastName)
+        nbaData.getTeamSchedule(teamId).then(resp => console.log(resp))
+      }
+    )
+  }
+  if (message.author.username !== client.user.username) {
+    if (message.channel.type === 'dm') {
+      const msg = message.content.split(' ')
+      if (msg[0] === 'setRefresh' && msg.length > 1) {
```
Comment (jacky-tran) [inline app.js]: Would there ever be a case where we would want the message length to be greater than 2? If not, we should guard against that. 
In either case though, we should also send back an error message if the input is incorrect.

**Review by jacky-tran — DISMISSED** [review]
Let's also try to get the build to pass on this branch.

File: app.js (hunk 2 above)
Comment (who03) [reply app.js]: I fix

**Review by Laturine — COMMENTED** [review]
looks good! Let's rebase and merge this.

File: app.js (hunk 1 above)
Comment (who03) [reply app.js]: Done

**Pull request merged**